cmd/ref_select.go

```diff
@@ -40,7 +41,14 @@ func NewEmptyRefSelect() *RefSelect {
 }
 
 // NewExplicitRefSelect returns a single explicitly provided reference
-func NewExplicitRefSelect(ref string) *RefSelect {
+func NewExplicitRefSelect(ref string, fsi *lib.FSIMethods) *RefSelect {
+	if fsi != nil {
+		dsRef := reporef.DatasetRef{}
+		err := fsi.FSIDatasetForRef(&ref, &dsRef)
+		if err == nil {
+			return &RefSelect{kind: "for linked", refs: []string{ref}, dir: dsRef.Path}	
```

Comment:
We do a check on `kind` to define an `explicit` ref and `dri` to check if it is a `linked` ref.